### ISSUE TITLE
Right button mouse single click setting is ignored with portable ini file if there's no single click context menu for any button

### ISSUE THREAD

**Issue by anonim1979** [White-Tiger/T-Clock#163]
Right button mouse single click setting is ignored with portable ini file if there's no single click context menu for any button. After quitting program it is not read back from ini file (whatever you've chosen is written to it) and always set itself to "context menu".

This is if "context menu" isn't set to some other "single click" button. So I can't set ex. double click middle button for context menu without my right button single click overwritten. 

Edit.
It seems registry behaves similar.

EDIT2.
As a workaround I set button 5 single click for context menu so I can set single clicks for l/r/m buttons as I wish.

Tested nightly, 2.4.4-RC and a few older releases. The same behaviour. 
Windows 8.1 x64

BTW. Would it be possible to localize context menu? Not the program itself (as I expect it would take a lot of effort). Or allow users to put at least the one most basic translations of "adjust time/date" etc. strings in the .ini / registry to be displayed instead of English names?

**Comment by White-Tiger** [comment]
you're right.. seems like the "does context menu exist" check at startup, differs from the one when configuring mouse actions...
https://github.com/White-Tiger/T-Clock/blob/2eb7e2aaf48af0b20bedaff5d5d629216011b382/src/Clock/PageMouse.c#L511
the second '0' was likely meant to be '1'
